src/components/LoginContent.js

```diff
@@ -0,0 +1,225 @@
+
+import EyeIcon from '../assets/images/icons/eye.png';
+import HideEyeIcon from '../assets/images/icons/invisible-2.png';
+
+if (
+    Platform.OS === "android" &&
+    UIManager.setLayoutAnimationEnabledExperimental
+) {
+    UIManager.setLayoutAnimationEnabledExperimental(true);
+}
+
+class LoginContent extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            isLoggedIn: true,
+            isInRegister: false,
+            hidePrimaryPassword: true,
+            hideConfirmationPassword: true,
+            isPasswordConfirmed: false,
+            isRegisterReady: false,
+        };
+
+        this.handleActionButton = this.handleActionButton.bind(this);
+        this.onRegisterLoginLinkClick = this.onRegisterLoginLinkClick.bind(this);
+        this.triggerPrimaryPasswordVisilibity = this.triggerPrimaryPasswordVisilibity.bind(this);
+        this.triggerConfirmationPasswordVisilibity = this.triggerConfirmationPasswordVisilibity.bind(this);
+        this.onForgotPasswordClick = this.onForgotPasswordClick.bind(this);
+        this.comparePasswords = this.comparePasswords.bind(this);
+    }
+
+    payload = {
+        fullName: this.fullName,
+        email: this.email,
+        password: this.password,
+    } 
+
+    onRegisterLoginLinkClick() {
+      // In order to animate the next state change
+      LayoutAnimation.configureNext(LayoutAnimation.Presets.linear);
+      this.setState({ isInRegister : !this.state.isInRegister });
+    }
+
+    onForgotPasswordClick() {
+      this.props.navigation.navigate('ForgotPasswordScreen');
+    }
+
+    handleActionButton() {
+        if (this.state.isInRegister) {
+            // Handle register functions
+            // JSON data to send when registering
+            let newUserCredentials = {
+                "name": this.payload.fullName,
+                "email": this.payload.email,
+                "password": this.payload.password
+            }
+            
+            axios
+                .post('http://34.125.37.12:8080/api/register', newUserCredentials)
+                .then(
+                    response => {
+                        console.log(response.data);
+                        this.props.navigation.navigate('RegisterScreen', {
+                            userPersonalEmail: this.payload.email,
+                          });
+                        // TODO: Save jwt in AsyncStorage
+                    }
+                )
+                .catch(
+                    error => {
+                        console.log(error)
+                    }
+                )
+                
+        } else {
+            // Handle login functions
+            // JSON data to send when logging in
+            let credentials = {
+                "email": this.payload.email,
+                "password": this.payload.password
+            }
+
+            axios
+                .post('http://34.125.37.12:8080/api/login', credentials)
```

Comment:
again, this URL has changed. This can be used from the constants.js file

Reply:
As mentionned earlier, global variables will be integrated later on, as they are not fully functionnal yet